docs/standard/serialization/system-text-json/source-generation.md

```diff
@@ -161,7 +163,7 @@ Here are the preceding examples in a complete program:
 
 ## Specify options by using `JsonSerializerOptions`
 
-Some options of <xref:System.Text.Json.JsonSerializerOptions> aren't supported by serialization optimization mode. Such options cause a fallback to the non-source-generated `JsonSerializer` code. For more information, see [Serialization optimization](source-generation-modes.md#serialization-optimization-mode).
+Some options of <xref:System.Text.Json.JsonSerializerOptions> aren't supported by serialization optimization mode, especially in .NET 7 and earlier versions. Such options cause a fallback to the non-source-generated `JsonSerializer` code. For more information, see [Serialization optimization](source-generation-modes.md#serialization-optimization-mode).
```

Comment:
Per https://github.com/dotnet/docs/pull/36549/files#r1286663329 this hasn't changed substantially in .NET 8.